code/game/objects/items.dm

```diff
@@ -587,7 +587,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 		var/turf/T = get_turf(src)
 		var/obj/effect/decal/cleanable/ash/A = new()

```

Comment:
It seems it's so that it can set the desc before it's insta killed.

Better question. If it starts off in nullspace, why isn't Entered being called when it moves to the turf?